File: Raven.Client.Java/src/main/java/net/ravendb/abstractions/data/SynchronizationDetails.java
package net.ravendb.abstractions.data;

import java.util.UUID;


public class SynchronizationDetails {
  private String fileName;
  private UUID fileETag;
  private String destinationUrl;
  private SynchronizationType type;

  public String getFileName() {
    return fileName;
  }

  public void setFileName(String fileName) {
    this.fileName = fileName;
  }

  public UUID getFileETag() {
    return fileETag;
  }

  public void setFileETag(UUID fileETag) {
    this.fileETag = fileETag;
  }

  public String getDestinationUrl() {
    return destinationUrl;
  }

  public void setDestinationUrl(String destinationUrl) {
    this.destinationUrl = destinationUrl;
  }

  public SynchronizationType getType() {
    return type;
  }

  public void setType(SynchronizationType type) {
    this.type = type;
  }

}

File: src/Raven.NewClient/FileSystem/SynchronizationDetails.cs
using Raven.NewClient.Abstractions.Data;

namespace Raven.NewClient.Abstractions.FileSystem
{
    public class SynchronizationDetails
    {
        public string FileName { get; set; }
        public long? FileETag { get; set; }
        public string DestinationUrl { get; set; }
        public SynchronizationType Type { get; set; }
    }
}
